Makefile.am

```diff
@@ -67,7 +67,8 @@ dist-hook: $(distdir)/tools/debian/copyright
 $(distdir)/tools/debian/copyright: $(DIST_STAMP)
 	$(AM_V_GEN) NODE_ENV=$(NODE_ENV) $(srcdir)/tools/build-debian-copyright > $@
 
-DISTCHECK_CONFIGURE_FLAGS = --enable-prefix-only
```

Comment:
Not 100% sure I agree with this change.  If the default upstream behaviour is to build Python, then checking that it installs/uninstals/etc. properly is actually pretty interesting...

Time for two separate distcheck scenarios now? :)

Reply:
Yeah, this is a bit ugly. I [previously tried to have that flag in the unit-tests container](https://github.com/cockpit-project/cockpit/compare/a7c98854000285eafbfe351621498cfabdd0fa58..c857e5351975b9f20cba4b3126a53fcd0e88f462), but that doesn't work, as distcheck doesn't copy the `./autogen.sh` options to the "inner" configure. With distchecking python, we both [get a build failure](https://github.com/cockpit-project/cockpit/actions/runs/5983773231/job/16234282838#step:4:840) which I don't understand (it works locally), and also we lose the coverage of the C bridge, which we actually want to keep.

If this can be parameterized from outside, that'd be better indeed, and we could also have two scenarios.

Reply:
I debugged the distcheck failure in a local container run, and the issue is that cockpit-{beiboot,askpass} are installed into ./cockpit-299+git/_inst/local/bin/cockpit-beiboot, but `mv` tries to move it from cockpit-299+git/_inst/bin/cockpit-askpass, i. e. missing the `/local` part. Somehow `$(bindir)` is missing the `$(prefix)`?

Reply:
Ah! It's https://bugs.debian.org/1035546 all over again. I originally had the workaround in Makefile.am, but @allisonkarlitskaya talked me into moving it to debian/rules. So I suppose this needs to move to Makefile.am after all :cry:

Reply:
I reworked this to fix distcheck with the Python bridge, and add a new distcheck scenario for the C bridge.